packages/frontend/src/components/ThemeProvider/index.tsx

```diff
@@ -18,7 +19,9 @@ const ThemeProvider = ({
   return (
     <ChakraThemeProvider theme={chakraTheme}>
       <MaterialThemeProvider theme={{ [THEME_ID]: materialTheme }}>
-        {children}
+        <Box display="flex" flexDir="column" minH="100vh">
```

Comment:
n00b qn - what happened to the age old practice of setting height = 100% in `html` and `body` tag

Reply:
then can set this to minH=100%